components/select_team/select_team.jsx

```diff
@@ -57,11 +60,36 @@ export default class SelectTeam extends React.Component {
         this.state = {
             loadingTeamId: '',
             error: null,
+            endofTeamsData: false,
+            currentListableTeams: [],
+            currentPage: 0,
         };
     }
 
     componentDidMount() {
-        this.props.actions.getTeams(0, TEAMS_PER_PAGE);
+        this.fetchMoreTeams();
+    }
+
+    UNSAFE_componentWillReceiveProps(nextProps) { // eslint-disable-line camelcase
+        this.setState(
+            {
+                currentListableTeams: [...nextProps.listableTeams],
+            }
+        );
+    }
+
+    fetchMoreTeams = async () => {
+        const {currentPage} = this.state;
+        const {actions} = this.props;
+
+        await actions.getTeams(currentPage, TEAMS_PER_PAGE, true);
```

Comment:
I think the true as third param is causing this API call to fail. Can you remove that?

Reply:
@AGMETEOR Aah ya, i need to check why it is not returning data on test server. I think there is an issue with the API

Reply:
The client error happens because the api is returning null. I raised a ticket to resolve this https://mattermost.atlassian.net/browse/MM-18119. We might have to wait for the fix

Reply:
The client error happens because the api is returning null. I raised a ticket to resolve this https://mattermost.atlassian.net/browse/MM-18119. We might have to wait for the fix

Reply:
@sudheerDev I refactored code not to pass true to `getTeams()`... I think we can spin up a test server and see now...